src/wrappers/any.jl

```diff
@@ -179,6 +179,14 @@ end
 
 Base.show(io::IO, status::Status) = print(io, ToString(status))
 
+# build a RayObject from a string
+function ray_object(data::String)
+    data_buffer = LocalMemoryBuffer(Ptr{Nothing}(pointer(data)), sizeof(data), true)
```

Comment:
I wonder if we need to do something here to keep the `data` from being GCed if it goes out of scope but the wrapped object does not...